database/seed/schema.cql

```diff
@@ -0,0 +1,35 @@
+DROP KEYSPACE IF EXISTS stock_chart;
+
+CREATE KEYSPACE stock_chart
+WITH replication = {
+  'class': 'SimpleStrategy',
+  'replication_factor' : 1
+};
+
+USE stock_chart;
+
+DROP TABLE IF EXISTS stock_quotes;
+DROP TABLE IF EXISTS stock_info;
+
+CREATE TABLE IF NOT EXISTS stock_info (
```

Comment:
Why do you need to specify "IF NOT EXISTS" here? Is it just a fail safe, in case the drop table command doesn't work for some reason?